charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.39.1
-version: 15.17.0
+version: 15.18.0
```

Because of label change its a major version change.

```suggestion
version: 16.0.0
```